src/test/java/org/sefglobal/scholarx/service/ProgramServiceTest.java

```diff
@@ -236,7 +236,7 @@ void getAppliedMentorsOfMentee_withUnavailableData_thenThrowNoContent() {
                 .findAllByProgramIdAndProfileId(anyLong(), anyLong());
 
```

Comment:
Update the first 3 lines to this:
```
        doReturn(Optional.empty())
                .when(menteeRepository)

I think it will fix fix the build.
                .findByProgramIdAndProfileId(anyLong(), anyLong());
```